package.json

```diff
@@ -19,7 +19,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/sundayempire/raygun4reactnative.git"
+    "url": "git+https://github.com/raygun4reactnative.git"
```

Comment:
This will need the organisations name within the URL. 

"git+https://github.com/MindscapeHQ/raygun4reactnative.git"

Reply:
Also I noticed the author.name being "name": "MindscapeHQ", which we should change to "name": "Raygun",

Reply:
Sweet I'll get onto it now